src/pages/article/Article.style.tsx

```diff
@@ -0,0 +1,136 @@
+import styled from 'styled-components';
+
+export const ProfileWrapper = styled.main`
+  padding: 0 50px 0 50px;
+`;
+
+export const User = styled.section`
+  display: flex;
+  margin-top: 25px;
+  justify-content: space-between;
+`;
+
+export const UserStatusWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 10px;
+`;
+
+export const Name = styled.span`
+  font-size: ${(props) => props.theme.fonts.$xs};
+  line-height: 16px;
+`;
+
+export const ActivityTime = styled.span`
+  font-size: ${(props) => props.theme.fonts.$xxs};
+  line-height: 13px;
+  color: #808080;
+`;
+
+export const ProgressSpan = styled.span`
+  font-size: ${(props) => props.theme.fonts.$xs};
+  max-width: fit-content;
+  background: #ebebeb;
+  border-radius: 10px;
+  padding: 5px 10px 5px 10px;
+`;
+
+export const ArticleDataWrapper = styled.section`
+  display: flex;
+  flex-direction: column;
+  margin-top: 25px;
+`;
+
+export const ArticleSection = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+`;
```

Comment:
중복코드인 것 같습니다.
```suggestion
export const ArticleSection = styled.div`
  display: flex;
  flex-direction: column;
  margin-top: 25px;
  gap: 10px;
`;
```
ArticleDataWrapper을 없애고 하나만 사용해도 같은 배치일 것 같아요